modelscope/pipelines/cv/rife_video_frame_interpolation_pipeline.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) Alibaba, Inc. and its affiliates.
+import glob
+import math
+import os
+import os.path as osp
+import subprocess
+import tempfile
+from typing import Any, Dict, Optional, Union
+
+import cv2
+import numpy as np
+import torch
+import torch.nn.functional as F
+from torchvision.utils import make_grid
+
+from modelscope.metainfo import Pipelines
+from modelscope.models.cv.video_frame_interpolation.rife import RIFEModel
+from modelscope.outputs import OutputKeys
+from modelscope.pipelines.base import Input, Pipeline
+from modelscope.pipelines.builder import PIPELINES
+from modelscope.preprocessors import LoadImage
+from modelscope.preprocessors.cv import VideoReader
+from modelscope.utils.config import Config
+from modelscope.utils.constant import ModelFile, Tasks
+from modelscope.utils.logger import get_logger
+
+logger = get_logger()
+
+
+@PIPELINES.register_module(
+    Tasks.video_frame_interpolation,
+    module_name=Pipelines.rife_video_frame_interpolation)
+class RIFEVideoFrameInterpolationPipeline(Pipeline):
```

Comment:
add pipeline doc str, refer to https://github.com/modelscope/modelscope/blob/master/modelscope/pipelines/cv/card_detection_correction_pipeline.py#L34C1-L34C1

Reply:
added